tests/baselines/reference/ModuleWithExportedAndNonExportedFunctions.errors.txt

```diff
@@ -1,4 +1,4 @@
-tests/cases/conformance/internalModules/exportDeclarations/ModuleWithExportedAndNonExportedFunctions.ts(28,13): error TS2339: Property 'fn2' does not exist on type 'typeof A'.
+tests/cases/conformance/internalModules/exportDeclarations/ModuleWithExportedAndNonExportedFunctions.ts(28,13): error TS2551: Property 'fn2' does not exist on type 'typeof A'. Did you mean 'fng'?

```

Comment:
We can actually adjust the algorithm to reflect this! We can just reduce the cost of an addition relative to a substitution, such that a suggestion with one character added is preferred to a suggestion with one character substituted.

The reason this suggestion was added, on inspection (since it doesn't use any transpositions), is because you weighted a substitution as 2 edits in your algorithm, whereas when I rewrote it I kept it at 1.  Should I just revert the weight of substitutions to 2?